### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mmiladi**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,3 +27,8 @@ script:
   - make test_ftp
   - make test_bioblend
   - make test_docker_in_docker
+
+# blocklist
+branches:
```
Comment (bgruening) [inline .travis.yml]: This is probably better: https://docs.travis-ci.com/user/customizing-the-build#Skipping-a-build

Comment (mmiladi) [reply .travis.yml]: Yes. I can't cancel the travis jobs. Is it because I am not admin in Backofenlab?

Comment (bgruening) [reply .travis.yml]: You are admin of this repo, so it should work. Are you login into travis?

Comment (bgruening) [reply .travis.yml]: Can you please revert these changes?

**Pull request merged**